_posts/2022-05-20-changelog-state-backend.md

```diff
@@ -338,6 +336,10 @@ We encourage you to try out this feature and assess the pros and cons of using i
 
 Please see the full documentation [here](https://nightlies.apache.org/flink/flink-docs-master/docs/ops/state/state_backends/#enabling-changelog).
 
+# Acknowledgments
+
+We thank Stephan Ewen and all the engineers who contributed to the project.
```

Comment:
LOL, maybe add a bit more names and descriptions of what are they thanked for ...

Otherwise, it is now reading like "we do not want to thank you"

For example,
We thank Stephan Ewen for the initial idea of the changelog, and many other engineers Piotr, Yu and Yun for design discussion and code review.

Reply:
I think this is rather a matter of taste 😁